packages/webview_flutter/webview_flutter_platform_interface/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0
+
+* Initial open-source release.
```

Comment:
This description doesn't really make sense; how about "Extracted platform interface from `webview_flutter`"

Reply:
Much better, thank you. I will pay more attention to this in the future.